sirepo/job_driver/docker.py

```diff
@@ -93,6 +93,7 @@ def init_class(cls, job_supervisor):
                 bool,
                 "mount ~/.pyenv, ~/.local and ~/src for development",
             ),
+            enable_gpus=(False, bool, "apply --gpus=all"),
```

Comment:
`"apply '--gpus=all'"`? Clearer?

Reply:
Make this identical to rsdockerspawner, which is what docker cli expects.

Reply:
@robnagler can you elaborate on what you'd like to see? [This is what I see](https://github.com/radiasoft/rsdockerspawner/blob/6783877fa9bb526a9da06e3cd6e4700d80e0e618/rsdockerspawner/rsdockerspawner.py#L105) in rsdockerspawner. When you set `--gpus=all` on the command line (as is done here) it sets the DeviceRequests config that rsdockerspawner sets itself.

Reply:
Thanks b0cbf7352